docs/debugging.md

```diff
@@ -5,13 +5,13 @@ Django Debug Toolbar (DDT)
 ----------------
 This is a useful tool for examining SQL queries, performance, headers, requests, signals, cache, logging, and more.  
 
-To enable DDT, you need to set your `INTERNAL_IPS` to the IP address of your load balancer.  This can be overriden in `local_settings`.  
+To enable DDT, you need to set your `INTERNAL_IPS` to the IP address of your load balancer.  This can be overriden by creating a new settings file beginning with `local_` in `awx/settings/` (e.g. `local_overrides.py`).
```

Comment:
nitpick, `overridden` instead of `overriden`.

Reply:
fine fine, even though that wasn't me...

Reply:
updated.